src/Microsoft.ML.AutoML/Utils/MLNetUtils/VectorUtils.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+namespace Microsoft.ML.AutoML
+{
+    internal static class VectorUtils
+    {
+        public static double GetMean(double[] vector)
+        {
+            double sum = 0;
+            for (int i = 0; i < vector.Length; i++)
+            {
+                sum += vector[i];
```

Comment:
Do we want to use `CpuMath` here so this is vectorized?

Reply:
Can't do it for this release due to test project dependency, should be done as part of https://github.com/dotnet/machinelearning/issues/4064